Python Quiz Game

```diff
@@ -0,0 +1,48 @@
+import random
+
+  print("Welcome to Q-Quiz!\nGood luck")
+
+  answer=input("Ready?")
+  score=0
+
+  if answer.lower()=='yes':
+      print("Great!")
+  else:
+      print("See you later!")
+
+  QUESTIONS = [
+      ('#question1', '#answer1'),
+      ('#question2', '#answer2'),
+      ('#question3', '#answer3'),
+  ]
+
+  num_questions=len(QUESTIONS)
+  questions = random.sample(list(QUESTIONS), k = num_questions)
+  num = random.randint(0,#n)
```

Comment:
Why is #n here can't it be num_questions?
Or Len of the other list you created just above?

Reply:
I actually only started python a few weeks ago, and am not familiar with guard clauses. I shall research and then rectify.

Reply:
While loop used for "Ready?" user input